main.jl

```diff
@@ -6,6 +6,11 @@ Pkg.add(PackageSpec(name="CoverageTools"))
 
 using CoverageTools
 
-pf = process_folder()
+directories = get(ENV, "INPUT_DIRECTORIES", "src")
+dirs = filter!(!isempty, split(directories, ","))
+for dir in dirs
+    isdir(dir) || error("directory \"$dir\" not found!")
+end
 
-LCOV.writefile("lcov.info", pf)
+pfs = map(process_folder, dirs)
```

Comment:
`process_folder` returns a `Vector{FileCoverage}`. Using `map` this way, wouldn't it return a `Vector{Vector{FileCoverage}}`? Shouldn't it be something like
```suggestion
pfs = mapreduce(process_folder, vcat, dirs)
```
instead (although I am not expert on `mapreduce` or `map` :wink:)?

Reply:
ah oops, good point!